README.md

```diff
@@ -4,7 +4,53 @@ This library is intended to be used in conjunction with the OpenTelemetry SDK an
 
 It is based on https://github.com/NovoNordisk-OpenSource/heimdall-templates-dotnet-microservice/blob/main/src/Heimdall.Templates.Dotnet.Microservice.Infrastructure/OpenTelemetry
 
-# Example
+# Examples
+Bifrost can be instrumented in two ways. Either by using the Builder Extensions or by using the Exporter Extensions
+## Open Telemetry Builder
+Using the Open Telemetry Builder extension is the simplest solution but also the least flexible. It is recommended to use the Exporter Extensions if you need more control over the configuration.
+
+By using the telemetry builder you'll get
+- `AspNetCore`, `HttpClient` and `"Azure.*"` traces auto instrumentation.
+- `HttpClient` metrics auto instrumentation.
+- `IncludeScopes` and `IncludeFormattedMessage` logs. 
+
+You can also add your own activity sources and meters.
+
+```csharp
+var bifrostOptions = new BifrostOptions
```

Comment:
This part seems to have reverted the syntax fix changes I did in my PR

Reply:
Upsie. @michny can you please review again?

Reply:
Looks good